outreach/podcast/ideas/structure.md

```diff
@@ -0,0 +1,34 @@
+# OpCode Podcast Structure Ideas*
+*Progress over Perfection
+
+## Overview of Show:
+
+1. Message by David Molina
+2. Welcome and Intro
+3. Table of Contents
+4. Hightlights and News?
+5. Interview with Guest
+    - Background and story
+    - How OpCode has helped?
+    - *Consider non opcode members/veterans/mil/milspouse who transitioned to tech without OpCode?
+    - Pre Interview: Talk about personal story, review what was most important for their story?
```

Comment:
Maybe have a form we can develop here? So it can be an email first to set expectation?

Reply:
A form could definitely help. Depending on the guest/co-host/other person, I thought a quick <5 min phone call might better prepare everyone for the longer discussion for the podcast. 

This is also assuming that guests on the podcasts would be coming in cold via the OC network or some other social media reach out.